cmake/Version.cmake

```
@@ -119,39 +119,41 @@ else(CLEMENTINE_VERSION_PRERELEASE)
 endif(CLEMENTINE_VERSION_PRERELEASE)
 
 # Add git revision
-if(FORCE_GIT_REVISION)
-  set(GIT_REV ${FORCE_GIT_REVISION})
-  set(GIT_INFO_RESULT 0)
-else(FORCE_GIT_REVISION)
-  find_program(GIT_EXECUTABLE git)
-  message(STATUS "Found git: ${GIT_EXECUTABLE}")
-
-  if(NOT GIT_EXECUTABLE-NOTFOUND)
-    execute_process(COMMAND ${GIT_EXECUTABLE} describe
-        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-        RESULT_VARIABLE GIT_INFO_RESULT
-        OUTPUT_VARIABLE GIT_REV
-        OUTPUT_STRIP_TRAILING_WHITESPACE)
-    if(NOT ${GIT_INFO_RESULT} EQUAL 0)
-      message(SEND_ERROR "git describe failed with code ${GIT_INFO_RESULT}: ${GIT_REV}")
+if(INCLUDE_GIT_REVISION)
+  if(FORCE_GIT_REVISION)
+    set(GIT_REV ${FORCE_GIT_REVISION})
+    set(GIT_INFO_RESULT 0)
+  else(FORCE_GIT_REVISION)
+    find_program(GIT_EXECUTABLE git)
+    message(STATUS "Found git: ${GIT_EXECUTABLE}")
+
+    if(NOT GIT_EXECUTABLE-NOTFOUND)
```

This still suffers from the problem I was attempting to fix in #7342. I.e. it shouldn't check for a "NOTFOUND" variable. That variable doesn't exist so this is `NOT FALSE` i.e. a tautology (always true, making the condition redundant).

It should check for the *existence* of `GIT_EXECUTABLE` instead:
```suggestion
    if(GIT_EXECUTABLE)
```

See #7342's explanation for more details.

---

This may well still cause a problem with certain builds, but it wasn't something that would ever be checked with the release tarballs I'm focusing on. With release tarballs, no git information will be included in the tarball, so INCLUDE_GIT_REVISION should be changed to OFF in those tarballs, and, if my changes are included, would skip over any of the git checking, never running into the check you mentioned.

Since release tarballs generated by GitHub don't include the .git/ folder, they will always fail with the current detection methods.

I haven't looked into the git detection at all since I am the maintainer of this package for SlackBuilds.org and they don't support git cloning when building packages. So your change might still be needed, but it was not looked at in this PR. That line only changed due to indentation when adding the `if(INCLUDE_GIT_REVISION)` statement before everything else.